diff --git a/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.d.ts b/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.d.ts
--- a/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.d.ts
+++ b/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.d.ts
@@ -82,8 +82,9 @@ declare const _default: {
     upload: {
         progress: {
             uploadingText: string;
             waitingText: string;
             failText: string;
             successText: string;
+            reloadText: string;
         };
     };
diff --git a/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.js b/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.js
--- a/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.js
+++ b/node_modules/tdesign-mobile-vue/cjs/_common/js/global-config/mobile/locale/ja_JP.js
@@ -93,8 +93,9 @@ var jaJP = {
   upload: {
     progress: {
       uploadingText: "\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9\u4E2D\u2026",
       waitingText: "\u5F85\u6A5F\u4E2D",
       failText: "\u5931\u6557\u3057\u307E\u3057\u305F",
-      successText: "\u6210\u529F\u3057\u307E\u3057\u305F"
+      successText: "\u6210\u529F\u3057\u307E\u3057\u305F",
+      reloadText: "\u518D\u8A66\u884C"
     }
   },
diff --git a/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.d.ts b/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.d.ts
--- a/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.d.ts
+++ b/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.d.ts
@@ -82,8 +82,9 @@ declare const _default: {
     upload: {
         progress: {
             uploadingText: string;
             waitingText: string;
             failText: string;
             successText: string;
+            reloadText: string;
         };
     };
diff --git a/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.mjs b/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.mjs
--- a/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.mjs
+++ b/node_modules/tdesign-mobile-vue/es/_common/js/global-config/mobile/locale/ja_JP.mjs
@@ -93,8 +93,9 @@ var jaJP = {
   upload: {
     progress: {
       uploadingText: "\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9\u4E2D\u2026",
       waitingText: "\u5F85\u6A5F\u4E2D",
       failText: "\u5931\u6557\u3057\u307E\u3057\u305F",
-      successText: "\u6210\u529F\u3057\u307E\u3057\u305F"
+      successText: "\u6210\u529F\u3057\u307E\u3057\u305F",
+      reloadText: "\u518D\u8A66\u884C"
     }
   },
diff --git a/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.d.ts b/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.d.ts
--- a/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.d.ts
+++ b/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.d.ts
@@ -82,8 +82,9 @@ declare const _default: {
     upload: {
         progress: {
             uploadingText: string;
             waitingText: string;
             failText: string;
             successText: string;
+            reloadText: string;
         };
     };
diff --git a/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.js b/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.js
--- a/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.js
+++ b/node_modules/tdesign-mobile-vue/esm/_common/js/global-config/mobile/locale/ja_JP.js
@@ -93,8 +93,9 @@ var jaJP = {
   upload: {
     progress: {
       uploadingText: "\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9\u4E2D\u2026",
       waitingText: "\u5F85\u6A5F\u4E2D",
       failText: "\u5931\u6557\u3057\u307E\u3057\u305F",
-      successText: "\u6210\u529F\u3057\u307E\u3057\u305F"
+      successText: "\u6210\u529F\u3057\u307E\u3057\u305F",
+      reloadText: "\u518D\u8A66\u884C"
     }
   },
diff --git a/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.d.ts b/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.d.ts
--- a/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.d.ts
+++ b/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.d.ts
@@ -82,8 +82,9 @@ declare const _default: {
     upload: {
         progress: {
             uploadingText: string;
             waitingText: string;
             failText: string;
             successText: string;
+            reloadText: string;
         };
     };
diff --git a/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.js b/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.js
--- a/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.js
+++ b/node_modules/tdesign-mobile-vue/lib/_common/js/global-config/mobile/locale/ja_JP.js
@@ -93,8 +93,9 @@ var jaJP = {
   upload: {
     progress: {
       uploadingText: "\u30A2\u30C3\u30D7\u30ED\u30FC\u30C9\u4E2D\u2026",
       waitingText: "\u5F85\u6A5F\u4E2D",
       failText: "\u5931\u6557\u3057\u307E\u3057\u305F",
-      successText: "\u6210\u529F\u3057\u307E\u3057\u305F"
+      successText: "\u6210\u529F\u3057\u307E\u3057\u305F",
+      reloadText: "\u518D\u8A66\u884C"
     }
   },
